Makefile

```
@@ -1,4 +1,6 @@
 GOBIN ?= ${GOPATH}/bin
+IMG   ?= tackle2-hub:latest
+CONTAINER_RUNTIME := docker
```

This might be an improvement over what annoyed @jmontleon ... at least it can be overridden.

```suggestion
CONTAINER_RUNTIME ?= $(shell command -v podman 2> /dev/null || echo docker)
```

---

The other option is to drop this and just have `podman-build` and `docker-build` targets.